docs/FAQ.md

```diff
@@ -25,6 +25,17 @@ by this command is very limited and you may want to look for other tools if you
 There are many excellent serial terminals to chose from. On Linux or macOS, you may already have [screen][screen]
 installed. On Windows, a good choice for command line usage is Plink, included with [PuTTY][putty].
 
+## How to change Monitor Configuration?
+
+[Configuration parameters][configuration parameters] of the monitor can be obtained by executing the following commad:
+
+`$ arduino-cli monitor -p <port> --describe`
+
+These parameters can be modified by passing a list of `<key>=<desiredValue>` pairs to the `--config` flag. For example,
+to change the baud rate of the serial monitor to 4800, the command would be the following:
```

Comment:
```suggestion
when using a serial port, the monitor baud rate can be set to 4800 with the following command:
```

Adjust wording to move the association of the `serial` protocol from the monitor to the port. This is intended to communicate the concept that the monitor is a general purpose tool that can be used with any arbitrary communication channel rather than being specific to serial ports.